Source/WebKit/WebProcess/Plugins/PDF/PDFPluginBase.mm

```diff
@@ -187,6 +187,13 @@
     return [m_pdfDocument isLocked];
 }
 
+void PDFPluginBase::notifySelectionChanged()
+{
+    if (!m_frame || !m_frame->page())
+        return;
+    m_frame->page()->didChangeSelection(*m_frame->coreLocalFrame());
```

Comment:
```
_frame->protectedPage()->didChangeSelection(*RefPtr { m_frame->coreLocalFrame() });
```
or
```
_frame->protectedPage()->didChangeSelection(*m_frame->protectedCoreLocalFrame());
```

if you add a `protectedCoreLocalFrame()` getter that returns a RefPtr.

Reply:
Per our smart pointer rules, you need to protect both:
1. objects you call a non-trivial member function on
2. parameters you pass to such functions

Reply:
Fixing shortly, thanks for the catch - didn't notice the unprotected use when I was shuffling things around.

Reply:
Fixed in [a3dd9df](https://github.com/WebKit/WebKit/commit/a3dd9dfcbfecae40efc594c4f407c43cf4fbcab9)